CONTRIBUTING.md

```diff
@@ -58,11 +59,19 @@ If you are still having issues or if your command prompt is throwing an error th
 Check to see where the Python you're running is located (`which python`). You should see something like `/Users/your_account_shortname/.pyenv/shims/python`.
 If you don't see something similar, you may have several versions of Python installed elsewhere (via Anaconda or Homebrew). If (and only if) you'd like to clear out any previous homebrew-based installs, type `brew uninstall --ignore-dependencies python3 && brew uninstall --ignore-dependencies python`.
 
-### Database commands.
+### Database migrations.
 
-- Create and Seed the Database: `pipenv run python manage.py create`
-- Delete, create, Seed the Database: `pipenv run python manage.py recreate`
-- Delete the Database: `pipenv run python manage.py drop`
```

Comment:
I think we will still need to have these commands documented in the readme. We still need to them to create the database right?

Reply:
We already replaced "pipenv run python manage.py create" with "pipenv run flask db create". The "flask db create", "flask db recreate" and "flask db drop" are still functional.

Reply:
That's right! Thanks.